src/groups/bmq/bmqp/bmqp_messageproperties.h

```diff
@@ -684,6 +696,7 @@ MessageProperties::setProperty(const bsl::string& name, const TYPE& value)
 
     p.d_length  = newPropValueLen;
     p.d_value   = value;
+    // This cannot have bsl::string_view' type.
```

Comment:
```suggestion
    // This cannot have `bsl::string_view` type.
```
Missing opening quote

Reply:
Fixed here https://github.com/bloomberg/blazingmq/pull/1021